modules/openy_features/openy_egym_iwatch/src/scss/openy_egym_iwatch.scss

```diff
@@ -0,0 +1,312 @@
+      grid-column-gap: 40px;
+      grid-row-gap: 80px;
+      padding-top: 80px;
+
+      .grid-item {
+        .inner {
+          h2 {
+            margin-top: 40px;
+            margin-bottom: 20px;
+            letter-spacing: -0.9px;
+          }
+
+          .d-block {
+            img {
+              width: 360px;
+              height: 360px;
+            }
+          }
+
+          .text {
+            p {
+              font-size: 14px;
+              line-height: 21px;
+            }
+
+            p:first-child {
+              font-size: 18px;
+              line-height: 27px;
+              margin-bottom: 20px;
+            }
+          }
+        }
+      }
+    }
+  }
+
+  .landing-content div:nth-of-type(3), .landing-content .wrapper-field-content div:nth-of-type(3) .paragraph--grid-content-egym {
+    grid-template-columns: 220px;
+
+    .grid-item {
+      .inner {
+        h2 {
+          margin-top: 20px;
+          font-size: 18px;
+          letter-spacing: -0.45px;
+          line-height: 27px;
+        }
+
+        .d-block {
+          display: flex;
+          justify-content: center;
+
+          img {
+            width: 90px;
+            height: 90px;
+          }
+        }
+
+        .text {
+          font-size: 12px;
+          line-height: 18px;
+
+          p:first-child {
+            font-size: 12px;
+            line-height: 18px;
+          }
+        }
+      }
+    }
+
+    @media (min-width: 1060px) {
+      grid-template-columns: 360px;
+
+      .grid-item {
+        .inner {
+          h2 {
+            margin-top: 20px;
+            font-size: 24px;
+            letter-spacing: -0.6px;
+          }
+
+          .d-block {
+            img {
+              width: 180px;
+              height: 180px;
+            }
+          }
+
+          .text {
+            font-size: 14px;
+            line-height: 21px;
+
+            p:first-child {
+              font-size: 14px;
+              line-height: 21px;
+            }
+          }
+        }
+      }
+    }
+  }
+
+  .landing-content {
+    .paragraph--type--simple-content:first-child {
+      text-align: center;
+      margin: 0 10px 40px;
+      padding-bottom: 0;
+      padding-top: 42px;
+      font-size: 14px;
+      line-height: 21px;
+
+      @media (min-width: 600px) {
+        margin-left: 34px;
+        margin-right: 34px;
+      }
+
+      @media (min-width: 1060px) {
+        font-size: 18px;
+        line-height: 27px;
+        padding-top: 82px;
+        margin-bottom: 80px;
+      }
+    }
+  }
+
+  .viewport {
+    background-color: white;
+    z-index: 0;
+    position: relative;
+
+    .landing-content {
+      position: relative;
+      background-color: white;
+    }
+  }
+
+  #block-openy-lily-content, #block-openy-rose-content {
+    .container {
+      background-color: white;
+    }
+
+    .banner-fixed {
+      margin-top: 0;
+    }
+  }
+
+  .pre-footer {
+    margin: -20px;
+  }
+}
+
+.openy_egym_iwatch.openy_rose {
```

Comment:
We have to be sure that websites that have another theme which is based on rose will have this class at body.